packages/fastify/src/constants.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable turbo/no-undeclared-env-vars */
```

Comment:
Do we need to declare any other global vars in turbo.json?
Happy to create a ticket and do a more thorough review if needed.

Reply:
I only added the ones I thought could actually change between builds. All the other ones will resolve at runtime set by the user. They don't come from us or the build tool.

So I think we don't need to any additional